res/css/views/messages/_ReactionsRowButton.scss

```diff
@@ -17,14 +17,11 @@ limitations under the License.
 .mx_ReactionsRowButton {
     display: inline-flex;
     line-height: $font-20px;
-    margin-right: 6px;
```

Comment:
Reactions row button does not always require another button ie. single reaction means only one button, therefore margin property should have not been declared by default.